docs/_static/rtd_theme_overrides.css

```diff
@@ -71,14 +103,206 @@ div.pkgnav ul li {
+.rst-content .wy-alert-info.caution .wy-alert-title,
+.rst-content .wy-alert-info.danger .wy-alert-title,
+.rst-content .wy-alert-info.error .wy-alert-title,
+.rst-content .wy-alert-info.hint .wy-alert-title,
+.rst-content .wy-alert-info.important .wy-alert-title,
+.rst-content .wy-alert-info.tip .wy-alert-title,
+.rst-content .wy-alert-info.warning .wy-alert-title,
+.rst-content .seealso .wy-alert-title,
+.rst-content .wy-alert-info.admonition-todo .wy-alert-title,
+.rst-content .wy-alert-info.admonition .wy-alert-title,
+.wy-alert.wy-alert-info .rst-content .admonition-title,
+.rst-content .wy-alert.wy-alert-info .admonition-title,
+.rst-content .note .admonition-title,
+.rst-content .wy-alert-info.attention .admonition-title,
+.rst-content .wy-alert-info.caution .admonition-title,
+.rst-content .wy-alert-info.danger .admonition-title,
+.rst-content .wy-alert-info.error .admonition-title,
+.rst-content .wy-alert-info.hint .admonition-title,
+.rst-content .wy-alert-info.important .admonition-title,
+.rst-content .wy-alert-info.tip .admonition-title,
+.rst-content .wy-alert-info.warning .admonition-title,
+.rst-content .seealso .admonition-title,
+.rst-content .wy-alert-info.admonition-todo .admonition-title,
+.rst-content .wy-alert-info.admonition .admonition-title {
+    background: #277CB4;
+}
+
+/* error, danger */
+
+.rst-content .danger .admonition-title,
+.rst-content .danger .wy-alert-title,
+.rst-content .error .admonition-title,
+.rst-content .error .wy-alert-title,
+.rst-content .wy-alert-danger.admonition-todo .admonition-title,
+.rst-content .wy-alert-danger.admonition-todo .wy-alert-title,
+.rst-content .wy-alert-danger.admonition .admonition-title,
+.rst-content .wy-alert-danger.admonition .wy-alert-title,
+.rst-content .wy-alert-danger.attention .admonition-title,
+.rst-content .wy-alert-danger.attention .wy-alert-title,
+.rst-content .wy-alert-danger.caution .admonition-title,
+.rst-content .wy-alert-danger.caution .wy-alert-title,
+.rst-content .wy-alert-danger.hint .admonition-title,
+.rst-content .wy-alert-danger.hint .wy-alert-title,
+.rst-content .wy-alert-danger.important .admonition-title,
+.rst-content .wy-alert-danger.important .wy-alert-title,
+.rst-content .wy-alert-danger.note .admonition-title,
+.rst-content .wy-alert-danger.note .wy-alert-title,
+.rst-content .wy-alert-danger.seealso .admonition-title,
+.rst-content .wy-alert-danger.seealso .wy-alert-title,
+.rst-content .wy-alert-danger.tip .admonition-title,
+.rst-content .wy-alert-danger.tip .wy-alert-title,
+.rst-content .wy-alert-danger.warning .admonition-title,
+.rst-content .wy-alert-danger.warning .wy-alert-title,
+.rst-content .wy-alert.wy-alert-danger .admonition-title,
+.wy-alert.wy-alert-danger .rst-content .admonition-title,
+.wy-alert.wy-alert-danger .wy-alert-title {
+    background: #e31704;
+}
+
+/* Generic admonition titles */
+
+.wy-alert-title, .rst-content .admonition-title {
+    background: #277CB4;
+}
```

Comment:
This only includes some of CSS from `sphinx_rtd_theme_ext_color` and not all of it, since I didn't want to override any of our other CSS customizations.  I think I also wanted to be able to customize the color schemes for admonitions to make sure there's still sufficient contrast with the hyperlink font colors.

I wondering if it'd be worth putting these customizations into a separate CSS file since they're (hopefully) temporary and they have a specific purpose.  Hm...

 - [ ] Make sure there's a note in here that these customizations should be dropped after the `sphinx_rtd_theme` defaults end up matching the WCAG guidelines.

Reply:
I would test it out, but I'm fairly confident our CSS will still override `sphinx_rtd_theme_ext_color` if added to `extensions` since Sphinx places our stylesheet as the top priority.

Reply:
After a little research, this is how I'd approach things...

1. create a new directory `docs/_static/css/` to house CSS files that modify the theme CSS file (i.e. that associated with `sphinx_rtd_theme`)
2. Move our CSS file `rtd_theme_overrides.css` into the newly crated directory and rename it `plasmapy.css`
3. Since `sphinx_rtd_theme_ext_color_contrast` is not released on PyPI, copy [their CSS file](https://github.com/AaltoSciComp/sphinx_rtd_theme_ext_color_contrast/blob/master/sphinx_rtd_theme_ext_color_contrast/_static/sphinx_rtd_theme_ext_color_contrast.css) into the newly created directory.  Obviously add a header to the file explaining and crediting where it came from.
4. Update the `setup()` function in our `conf.py` to add both CSS files using `app.add_css_file()`.  The [documentation for `add_css_file()`](https://www.sphinx-doc.org/en/master/extdev/appapi.html#sphinx.application.Sphinx.add_css_file) function show you can set priorities for the added CSS files; thus, we can ensure our CSS always takes priority.

Reply:
I like!

Reply:
Done, except that I still need to...
 - [ ] Re-check the documentation to make sure that everything is working as intended. 

I don't think there were any conflicts that would make if necessary for me to set CSS file priorities.